### PR TITLE
Docs: clarify node bootstrap

### LINKED ISSUE #1595: Docs: clarify node bootstrap 101
People keep thinking they need to create a node DID before connecting to a network, which causes sync issues:

```
time="2022-10-27T08:25:43Z" level=warning msg="Rolling back transaction application due to error" error="root transaction already exists"
```

"Setting up your node" (https://nuts-node.readthedocs.io/en/stable/pages/getting-started/3-configure-your-node.html) docs could be more clear, with a numbered list of steps, e.g.:

1. Configure bootstrap node address
2. Start your node
3. Let it synchronize
4. Create node DID and configure it
5. Restart
6. Verify

Also add the "root transaction already exists" to https://nuts-node.readthedocs.io/en/stable/pages/integrating/faq-errors.html

### PR DESCRIPTION
**Pull request by reinkrul**
Closes #1595

### CODE REVIEW

**Review by gerardsn — CHANGES_REQUESTED** [review]
The issue suggest emphasizing step 3 to prevent the `root transaction already exists` issue.
IMO this fails to achieve that. Maybe also add a section or a note between `Configuring` and `Node Identity`?

**Comment by gerardsn** [comment]
> Maybe also add a section or a note between `Configuring` and `Node Identity`?
Or perhaps update the node that is already there. (3-configure-your-node.rst#L95)

**Review by gerardsn — CHANGES_REQUESTED** [review]
Good improvement of the documentation. Some minor things left

File: docs/pages/getting-started/3-configure-your-node.rst [hunk 1]
```diff
@@ -70,44 +85,74 @@ These transactions contain endpoints of other nodes. After a reboot, your node w
...
 2. Configure TLS using ``tls.certfile``, ``tls.certkeyfile`` and ``tls.truststorefile``.
 
 See :ref:`configuration reference <nuts-node-config>` for a detailed explanation on how to exactly configure the Nuts node.
 
-
 .. note::
 
-    You can start the node without configuring the network, but it won't connect and thus exchange data with other nodes. You'll have a private network with one single node. Perfect for local development, but a bit lonely.
+    You can start the node without configuring the network, but it won't connect and thus exchange data with other nodes.
+    You'll have a private network with one single node. Perfect for local development, but a bit lonely.
```
[the remark this answers is not in the document]
Comment (reinkrul) [reply docs/pages/getting-started/3-configure-your-node.rst:96]: yes

File: docs/pages/getting-started/3-configure-your-node.rst [hunk 2]
```diff
@@ -70,86 +87,101 @@ These transactions contain endpoints of other nodes. After a reboot, your node w
...
+- configure TLS using ``tls.certfile``, ``tls.certkeyfile`` and ``tls.truststorefile``
 
-1. Configure the bootstrap nodes using ``network.bootstrapnodes``.
-2. Configure TLS using ``tls.certfile``, ``tls.certkeyfile`` and ``tls.truststorefile``.
-
-See :ref:`configuration reference <nuts-node-config>` for a detailed explanation on how to exactly configure the Nuts node.
-
-
-.. note::
+If you're using a YAML file to configure your node, the following snippet how to configure these properties:
```
Comment (gerardsn) [inline docs/pages/getting-started/3-configure-your-node.rst:93]:
```suggestion
If you're using a YAML file to configure your node, the following snippet shows how to configure these properties:
```

**Review by gerardsn — APPROVED** [review]
There are a lot of lists on this [page](https://nuts-node.readthedocs.io/en/docs-clarify-node-bootstrap/pages/getting-started/3-configure-your-node.html), but this a good update I'd say.

**Pull request merged**